src/corehost/cli/bdl_file_entry.cpp

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I was trying to follow the convention similar to `fx_`... abbreviation for frameworks. 
I'll remove the `bdl` prefix and make a new namespace.

Reply:
I see - I guess it makes sense. On the other hand FX is a very common abbreviation, "bdl" would be new. My personal preference is to not use abbreviations unless they are very obvious. So if I would start over I would probably even use `framework` instead of `fx`.
But in this particular case the namespace feels like the best solution - it's a very separate piece of functionality...